src/main/kotlin/dev/binclub/paperbin/transformers/AntiNetherRoof.kt

```diff
@@ -179,14 +182,14 @@ object AntiNetherRoof: PaperBinFeature {
 		val x = location.x.toInt()
 		val y = location.y.toInt()
 		val z = location.z.toInt()
-		
+
 		if (x >= worldBorder || z >= worldBorder || x <= negWorldBorder || z <= negWorldBorder) {
 			return false
 		}
 		
-		when (location.world.environment) {
+		when (location.world.environment!!) {

```

Comment:
Is this null check necessary? Even if the variable is never null it will reduce performance by adding an extra branch that is already handled by the switch

Reply:
Enum argument can be null in Java, but exhaustive when contains no null branch

Reply:
The when is not exhaustive, it does not provide any return value so it does not matter if none of its branches are triggered

Reply:
Good to know, I'll make the change